src/Client.ts

```diff
@@ -308,6 +309,10 @@ export class Client extends EventEmitter implements IClient {
         throw new PermissionDeniedError('deleteGroup', 'Participant');
     }
 
+    public deleteScene(_: ISceneDeletionParams): Promise<void> {
```

Comment:
Btw you don't actually have to take arguments to implement an interface. E.g. this is fine

```ts
interface IFoo {
    bar(x: number);
}

class Foo implements IFoo {
    bar() { }
}
```